README.md

```diff
@@ -110,6 +110,13 @@ The nodes are deployed in multiple regions around the globe using multi-cloud, h
 
 Checkout: [chainstack.com](https://chainstack.com/)
 
+### Node RPC
```

Comment:
to add content here you have to be a sponsor. For more info email [sponsor@multinodes.xyz](mailto:sponsor@multinodes.xyz)

Reply:
ahh, okay, I've removed the description for now